libvast/vast/plugin.hpp

```diff
@@ -232,18 +232,27 @@ class writer_plugin : public virtual plugin {
 /// @relates plugin
 class plugin_ptr final {
 public:
+  /// The display policy of a plugin.
+  enum class visible {
+    yes, ///< Show the plugin in the output of the version command.
+    no,  ///< Hide the plugin in the output of the version command.
+  };
```

Comment:
It feels a bit weird to have the plugin dictate how `vast version` behaves. What do you think about an enum like this

```
enum class plugin_type {
  dynamic,
  static,
  native
}
```

and then letting `vast version` decide itself how it renders each type of plugin?

Reply:
I made this change, please check back.